lesson-7/todos/src/components/Footer.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import '../styles/Footer.css'
+
+class Footer extends React.Component {
```

Comment:
Do we want to use stateless functional components?

Reply:
Cool idea. I think we haven't mentioned them yet. So it might be an interesting point. To mention React.Component vs React. PureComponent vs functional component.